mne_bids/report.py

```diff
@@ -501,7 +498,7 @@ def make_report(bids_root, session=None, verbose=True):
     kinds = get_kinds(bids_root)
 
     # only summarize allowed kinds (MEEG data)
-    kinds = [kind.upper() for kind in kinds if kind in ALLOWED_KINDS]
+    kinds = [kind.upper() for kind in kinds if kind in ALLOWED_MODALITY_KINDS]
```

Comment:
Just spotted this… this would produce `IEEG` instead of `iEEG`, right? not so pretty!